source-code/sdk-js/src/helpers/ast.test.ts

```diff
@@ -25,22 +24,22 @@ describe("ast - wrapVariableDeclaration", () => {
             })
         `
 		const inputAst = parse(input, acornOptions) as unknown as Program
-		const resultAst = await wrapVariableDeclaration(inputAst, "load", "wrapFn")
+		const resultAst = wrapVariableDeclaration(inputAst, "load", "wrapFn")[0]
```

Comment:
```suggestion
		const [resultAst] = wrapVariableDeclaration(inputAst, "load", "wrapFn")
```

and for all following function calls